modules/openapi-generator-online/Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM openjdk:8-jre-alpine
-  
+
 WORKDIR /generator
 
-COPY target/openapi-generator-online-3.0.0-SNAPSHOT.jar /generator/openapi-generator-online.jar
+COPY target/*.jar /generator/openapi-generator-online.jar
```

Comment:
@cbornet I tried *.jar before but I got errors and that's why I used the full filename instead.

Reply:
That's weird. Works on my machine 😉 . And that's also what we do in JHipster.
Did you do `mvn clean` before packaging ?

Reply:
(I also tested *.jar locally before and worked fine for me)

Reply:
Here is the error reported by the CI (travis):
```
Removing intermediate container eaef9e5f5b5b
Step 3/6 : COPY target/*.jar /generator/openapi-generator-online.jar
When using COPY with more than one source file, the destination must be a directory and end with a /
```
Ref: https://travis-ci.org/OpenAPITools/openapi-generator/builds/379603374

I'll submit a PR to fix it.

Reply:
So it seems there are multiple jar in the target directory. That shouldn't be the case, isn't it ?
Can you try `COPY target/openapi-generator-online*.jar /generator/openapi-generator-online.jar` so there's no need to modify the Dockerfile for each new version ?

Reply:
Same result:
```
Removing intermediate container 5b368abf32be
Step 3/6 : COPY target/openapi-generator-online-*.jar /generator/openapi-generator-online.jar
When using COPY with more than one source file, the destination must be a directory and end with a /
```

Let's go with the hard-coded solution for the time being as it's not something we want to automate with the highest priority.